omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -691,7 +691,10 @@
 # allowable choices come from the PGSSLMODE environment var for libpq.
 # supported sslmode values: disable | require
 default['private_chef']['postgresql']['sslmode'] = 'disable'
-
+# for now, copy these by hand into the directory, `chmod 0600 *.key; chown root *.key`
+# later, this will be automated and these comments will be removed
+default['private_chef']['postgresql']['ssl_cert_file'] = "#{default['private_chef']['postgresql']['data_dir']}/api.chef-server.dev.crt"
+default['private_chef']['postgresql']['ssl_key_file']  = "#{default['private_chef']['postgresql']['data_dir']}/api.chef-server.dev.key"
```

Comment:
would the `api.chef-server.dev.key` and `api.chef-server.dev.crt` exist by default?

Reply:
Yes, my idea was to have a recipe place those files by default (similar to nginx certificates existing by default).

Reply:
By the way, maybe I've forgotten the rationale.  Why do we want or need custom-nameable and custom-locateable certs (ssl_cert_file and ssl_key_file) for internal postgresql?  What's the use case?

Reply:
To answer my own question above, I found this from @markan lurking in my notes:
```
I think there’s two use cases to cover
1) Self signed certs of some sort; by default I think we generate one for nginx, we should reuse it for postgres.
2) Externally generated certs (the user has a cert for their host generated however they like)
In both cases we probably should follow the path that we use for nginx, and the same certs should be used for both nginx and postgres.
```